src/Benchmarks/Benchmarks/DeserializeFromJsonBodyBenchmark.cs

```diff
@@ -42,40 +39,37 @@ public void GlobalSetup()
 
             var gqlRequest = new GraphQLRequest { Query = SchemaIntrospection.IntrospectionQuery };
             var gqlRequestJson = Serializer.ToJson(gqlRequest);
-            _httpRequest = GetHttpRequestFor(gqlRequestJson);
+            _httpRequestBody = GetHttpRequestBodyFor(gqlRequestJson);
 
             gqlRequest.OperationName = "someOperationName";
-            gqlRequest.Inputs = new GraphQLSerializer().Deserialize<Inputs>(SHORT_JSON);
+            gqlRequest.Variables = new StjDeserializer().Deserialize<Inputs>(SHORT_JSON);
             var gqlRequestJson2 = Serializer.ToJson(gqlRequest);
-            _httpRequest2 = GetHttpRequestFor(gqlRequestJson2);
+            _httpRequestBody2 = GetHttpRequestBodyFor(gqlRequestJson2);
         }
 
-        private static HttpRequest GetHttpRequestFor(string gqlRequestJson)
+        private static Stream GetHttpRequestBodyFor(string gqlRequestJson)
         {
-            var httpContext = new DefaultHttpContext();
-            httpContext.Features.Set<IRequestBodyPipeFeature>(new RequestBodyPipeFeature(httpContext));
-            httpContext.Request.Body = new MemoryStream(Encoding.UTF8.GetBytes(gqlRequestJson));
-            return httpContext.Request;
+            return new MemoryStream(Encoding.UTF8.GetBytes(gqlRequestJson));
         }
 
         [IterationSetup]
         public void IterationSetup()
         {
             // Reset stream positions
-            _httpRequest.Body.Position = 0;
-            _httpRequest2.Body.Position = 0;
+            _httpRequestBody.Position = 0;
+            _httpRequestBody2.Position = 0;
         }
 
         [Benchmark(Baseline = true)]
-        public Task<GraphQLRequestDeserializationResult> NewtonsoftJson() => _nsjDeserializer.DeserializeFromJsonBodyAsync(_httpRequest);
+        public ValueTask<GraphQLRequest[]> NewtonsoftJson() => _nsjDeserializer.ReadAsync<GraphQLRequest[]>(_httpRequestBody);
```

Comment:
I suggest to merge this file into `DeserializationBenchmark` from main repo and completely delete file from here. Nothing ASP.NET related to bench here.

Reply:
#718